packages/backend-api-client/src/models/components/user.ts

```diff
@@ -0,0 +1,485 @@
+    backup_code_enabled: z.boolean().optional(),
+    mfa_enabled_at: z.nullable(z.number().int()).optional(),
+    mfa_disabled_at: z.nullable(z.number().int()).optional(),
+    external_accounts: z.array(ExternalAccountWithVerification$inboundSchema)
+      .optional(),
+    saml_accounts: z.array(SAMLAccount$inboundSchema).optional(),
+    last_sign_in_at: z.nullable(z.number().int()).optional(),
+    banned: z.boolean().optional(),
+    locked: z.boolean().optional(),
+    lockout_expires_in_seconds: z.nullable(z.number().int()).optional(),
+    verification_attempts_remaining: z.nullable(z.number().int()).optional(),
+    updated_at: z.number().int().optional(),
+    created_at: z.number().int().optional(),
+    delete_self_enabled: z.boolean().optional(),
+    create_organization_enabled: z.boolean().optional(),
+    create_organizations_limit: z.nullable(z.number().int()).optional(),
+    last_active_at: z.nullable(z.number().int()).optional(),
+    legal_accepted_at: z.nullable(z.number().int()).optional(),
+    primary_email_address: z.nullable(EmailAddress$outboundSchema).optional(),
+    primary_phone_number: z.nullable(PhoneNumber$outboundSchema).optional(),
+    primary_web3_wallet: z.nullable(Web3Wallet$outboundSchema).optional(),
+    full_name: z.nullable(z.string()).optional(),
+  }).transform((v) => {
+    const remappedObj = remap$(v, {
+      "external_id": "externalId",
+      "primary_email_address_id": "primaryEmailAddressId",
+      "primary_phone_number_id": "primaryPhoneNumberId",
+      "primary_web3_wallet_id": "primaryWeb3WalletId",
+      "first_name": "firstName",
+      "last_name": "lastName",
+      "profile_image_url": "profileImageUrl",
+      "image_url": "imageUrl",
+      "has_image": "hasImage",
+      "public_metadata": "publicMetadata",
+      "private_metadata": "privateMetadata",
+      "unsafe_metadata": "unsafeMetadata",
+      "email_addresses": "emailAddresses",
+      "phone_numbers": "phoneNumbers",
+      "web3_wallets": "web3Wallets",
+      "password_enabled": "passwordEnabled",
+      "two_factor_enabled": "twoFactorEnabled",
+      "totp_enabled": "totpEnabled",
+      "backup_code_enabled": "backupCodeEnabled",
+      "mfa_enabled_at": "mfaEnabledAt",
+      "mfa_disabled_at": "mfaDisabledAt",
+      "external_accounts": "externalAccounts",
+      "saml_accounts": "samlAccounts",
+      "last_sign_in_at": "lastSignInAt",
+      "lockout_expires_in_seconds": "lockoutExpiresInSeconds",
+      "verification_attempts_remaining": "verificationAttemptsRemaining",
+      "updated_at": "updatedAt",
+      "created_at": "createdAt",
+      "delete_self_enabled": "deleteSelfEnabled",
+      "create_organization_enabled": "createOrganizationEnabled",
+      "create_organizations_limit": "createOrganizationsLimit",
+      "last_active_at": "lastActiveAt",
+      "legal_accepted_at": "legalAcceptedAt",
+    });
+
+    return {
+      ...remappedObj,
+      get primaryEmailAddress() {
```

Comment:
Custom getters to better align with our existing API resources

Reply:
This specific file has been ignored during generation. We'll need to keep this in mind when regenerating in the future.